### PR TITLE
Update the Android installation instructions

### PR DESCRIPTION
**Pull request by felipecsl**
This seems to be causing a lot of confusion

@lelandrichardson @christopherdro

### CODE REVIEW

**Comment by christopherdro** [comment]
We should still leave the old setup and label the new one with `For React Native >= v0.29`. Similiar to how code push has their docs setup. 
https://github.com/Microsoft/react-native-code-push#plugin-configuration-android

**Comment by felipecsl** [comment]
Fair enough. I added the old instructions back

**Comment by christopherdro** [comment]
Maybe its worth it to also specify what version supports RN 29 somewhere in the docs?

**Comment by felipecsl** [comment]
Yes, that's definitely a good idea. I think it should go in the `CHANGELOG.md` with the release notes

File: docs/installation.md [hunk 1]
```diff
@@ -40,81 +40,71 @@ To install using Cocoapods, simply insert the following line into your `Podfile`
...
+1. in your `build.gradle` add:
+```groovy
+...
+dependencies {
+  ...
+  compile 'com.airbnb.android:react-native-maps:0.7.1'
+}
+```
+
+3. in your application object, add:
```
Comment (ahmed1490) [inline docs/installation.md]: number 2 is missing

**Pull request merged**